modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+//
+// * virt/storage/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling volume snapshot boot sources
+
+Enable volume snapshot boot sources by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
+
+[NOTE]
+====
+It is recommended to use volume snapshots on a storage profile that is proven to scale better when cloning from a single snapshot.
```

Comment:
We're typically supposed to [direct users to take a recommended action rather than stating something is recommended](https://redhat-documentation.github.io/supplementary-style-guide/#recommend), though up to you if you feel strongly about keeping it:

```suggestion
Use volume snapshots on a storage profile that is proven to scale better when cloning from a single snapshot.
```